pkgs/top-level/all-packages.nix

```diff
@@ -1783,6 +1783,8 @@ with pkgs;
 
   sitespeed-io = callPackage ../tools/networking/sitespeed-io { };
 
+  slipstream = callPackage ../tools/games/slipstream { };
```

Comment:
```suggestion
  slipstream = callPackage ../tools/games/slipstream {
    jdk = jdk8;
  };
```

Reply:
I was considering this too, but is it really necessary? I've tested Slipstream and it's both unmaintained and only works with jdk8. Or is it just a general pattern that's good to follow? I was under the impression it was mainly used for having multiple versions of a package (i.e. `something` and `something-qt`). Sorry if my questions are stupid, I hope it's okay to ask! :)

Reply:
I've decided to do it anyhow - I'm guessing it's because it makes it easier to override? (read: possible ...). Not sure how I missed that but thank you anyway :)